content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,237 @@
+is a `ValidatingWebhookConfiguration` in an example `istio-webhook-config.yaml` (only
+a part of the configuration is shown to save space).
+
+    {{< text yaml >}}
+    apiVersion: admissionregistration.k8s.io/v1beta1
+    kind: ValidatingWebhookConfiguration
+    metadata:
+      name: istio-galley
+      labels:
+        app: galley
+        chart: galley
+        heritage: Tiller
+        release: istio
+        istio: galley
+    webhooks:
+      - name: pilot.validation.istio.io
+        clientConfig:
+          service:
+            name: istio-galley
+            namespace: istio-system
+            path: "/admitpilot"
+          caBundle: ""
+        rules:
+          - operations:
+            - CREATE
+            - UPDATE
+            apiGroups:
+            - config.istio.io
+            ... SKIPPED
+        failurePolicy: Fail
+        sideEffects: None
+    {{< /text >}}
+
+1.  Use `istioctl` to enable the webhook configurations:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook enable --webhook-secret dns.istio-galley-service-account \
+        --namespace istio-system --validation-path galley-webhook.yaml \
+        --injection-path sidecar-injector-webhook.yaml
+    {{< /text >}}
+
+1.  To check that the sidecar injector webhook is working, verify that the webhook injects a
+sidecar container into an example pod with the following commands:
+
+    {{< text bash >}}
+    $ kubectl create namespace test-injection; kubectl label namespaces test-injection istio-injection=enabled
+    $ kubectl run --generator=run-pod/v1 --image=nginx nginx-app --port=80 -n test-injection
+    $ kubectl get pod -n test-injection
+    {{< /text >}}
+
+    The output from the `get pod` command should show the following output. The `2/2` value means that
+    the webhook injected a sidecar into the example pod:
+
+    {{< text plain >}}
+    NAME    READY   STATUS    RESTARTS   AGE
+    nginx-app   2/2     Running   0          10s
+    {{< /text >}}
+
+1.  Check that the validation webhook is working:
+
+    {{< text bash >}}
+    $ kubectl create namespace test-validation
+    $ kubectl apply -n test-validation -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: invalid-gateway
+    spec:
+      selector:
+        # DO NOT CHANGE THESE LABELS
+        # The ingressgateway is defined in install/kubernetes/helm/istio/values.yaml
+        # with these labels
+        istio: ingressgateway
+    EOF
+    {{< /text >}}
+
+    The output from the gateway creation command should show the following output. The error
+    in the output indicates that the validation webhook checked the gateway's configuration YAML file:
+
+    {{< text plain >}}
+    Error from server: error when creating "invalid-gateway.yaml": admission webhook "pilot.validation.istio.io" denied the request: configuration is invalid: gateway must have at least one server
+    {{< /text >}}
```

Comment:
same comment here, i would expect this to fail in the default install also.

Reply:
need more wording to explain why this is different and how to detect this is running in this new mode.

Reply:
@linsun How this is different does not belong in the task. We can work on a blog post evaluating both approaches but let's try not to overburden readers with information here.

Reply:
Added a step to verify that there are no existing webhook configurations for Galley and the sidecar injector before running the webhook enable command.